### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dagobertrenouf**
Seems to me this was forgotten when changing the name of the projet.

### CODE REVIEW

**Comment by swalkinshaw** [comment]
It wasn't named `ansible` because the project was `bedrock-ansible`.

But this does bring up a good point of whether this should be renamed to `trellis` anyway.

**Comment by dagobertrenouf** [comment]
I'm coming from using bedrock-ansible previously, it jumped on me like as something that had been forgotten while changing names.

**Comment by dagobertrenouf** [comment]
I remember in an earlier version of the repo the instructions said:

You should now have the following directories at the same level somewhere:
`- bedrock-ansible/
- example.dev/`

I think that's why I associate the folder name with the name of the repo.

**Comment by retlehs** [comment]
if it's changed to `trellis` then it seems like `site` should be renamed to `bedrock`

i say keep it as is right now

**Comment by mAAdhaTTah** [comment]
@retlehs If that's the case, maybe changing the name to `server` (so it's `server` & `site`) might make more sense?

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You should now have the following directories at the same level somewhere:
 
 ```
 example.com/    - Primary folder for the project
-├── ansible/    - Your version of this repo (renamed to just `ansible`)
+├── trellis/    - Your version of this repo (renamed to just `trellis`)
```
Comment (austinpray) [inline README.md]: While we are at it: 

"Your version of this repo (renamed to just `trellis`)" :arrow_right: "Your clone of this repo"

Comment (swalkinshaw) [inline README.md]: @dagobertrenouf are you able to update this?

Comment (dagobertrenouf) [inline README.md]: Yep, will do over the weekend

Comment (dagobertrenouf) [inline README.md]: Will only be able to do this week after all.

**Comment by dagobertrenouf** [comment]
@swalkinshaw added @austinpray suggestion. Let me know if I need to do something more (like squish all commits in one? I'm not sure).

**Comment by swalkinshaw** [comment]
@dagobertrenouf thanks. Squash would be great :+1:

**Comment by dagobertrenouf** [comment]
@swalkinshaw done ;)

**Comment by swalkinshaw** [comment]
Thanks!

**Pull request merged**